src/functions/questionnaire/create-quiz/index.ts

```diff
@@ -0,0 +1,13 @@
+import { handlerPath } from "@libs/handler-resolver";
+
+export default {
+  handler: `${handlerPath(__dirname)}/handler.main`,
+  events: [
+    {
+      httpApi: {
+        method: "post",
+        path: "/admin/questionnaire"
```

Comment:
```suggestion
        path: "/questionnaire",
          authorizer: {
          name: "timebankKeycloakAuthorizer"
        }
```

Reply:
Need to add the authorizer as this is what allows the logged in user in metatavu home to be authorized based on the keycloak access token.